uui-editor/src/implementation/Toolbar.tsx

```diff
@@ -16,19 +17,33 @@ interface ToolbarProps {
     placement?: 'top' | 'bottom' | 'right' | 'left' | 'auto';
 }
 
+export const getSelectionBoundingClientRect = (editor: PlateEditor): ClientRectObject => {
+    const domSelection = window.getSelection();
+
+    if (!domSelection || domSelection.rangeCount < 1) {
+        return getDefaultBoundingClientRect();
+    }
+
+    const domRange = domSelection.getRangeAt(0);
+
+    const clientRect = domRange.getBoundingClientRect();
+    if(clientRect.bottom === 0 && clientRect.right === 0) {
+        return domSelection.anchorNode.parentElement.getBoundingClientRect();
+    }
```

Comment:
This is the only difference between **[util](https://github.com/udecode/plate/blob/main/packages/floating/src/utils/getSelectionBoundingClientRect.ts)** provided by plate

Reply:
At some point on clicking empty cell in the table, `window.getSelection()` starts to select `<span>` instead of `text` which leads to `getBoundingClientRect()` returns 0,0,0,0 for each of dimentions.

Reply:
Here we selecting parent element of current selection and getting `getBoundingClientRect()` which returns valid dimentions.

Reply:
Discussed offline